bin/alluxio-common.sh

```diff
@@ -36,7 +36,8 @@ function get_alluxio_property() {
   echo "${property}"
 }
 
-# Generates an array of ramdisk paths in global variable RAMDISKARRAY 
+# Generates an array of ramdisk paths in global variable RAMDISKARRAY
+# - Only examines the top-tier worker cache level for "MEM"-type paths
```

Comment:
My current understanding is that there is no reason why we _couldn't_ specify `MEM` as the medium type for lower tiers, other than this function `get_ramdisk_array()` not checking for them.
- As a result, `alluxio-mount.sh` would not mount those ramdisk paths in lower tiers

Reply:
Can your documentation be a little more specific since this is just a comment. e.g.
```suggestion
# - Only examines level0 of the tiered store for "MEM"-type paths
```